hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenGrantKycToAccountHandlerTest.java

```diff
@@ -162,12 +165,29 @@ class HandleTests {
         @Mock
         private WritableTokenRelationStore tokenRelStore;
 
+        @Mock
+        private ReadableTokenStore readableTokenStore;
+
+        private static final AccountID TREASURY_ACCOUNT_9876 = BaseCryptoHandler.asAccount(9876);
+        private static final TokenID TOKEN_531 = BaseTokenHandler.asToken(531);
+
+        private static final Token newToken531() {
```

Comment:
Sorry for not being clear. I actually meant something like this:

```
private static final Token MY_TOKEN = Token.newBuilder()...
```

But it does not hurt and can be fixed later.

Reply:
Okay, will fix them in the next PR :pray: .

Reply:
Fixed these as well.

Reply:
@netopyr one last approve please :pray: